docs/guide/notifications.md

```diff
@@ -5,17 +5,32 @@ Here is a full list of curated Cirrus Actions for GitHub including ones to send
 
 ### Email Action
 
-Email GitHub Action allows to send email notifications on Cirrus CI Checks completion. Simply add the following to you
-`.github/main.workflow` workflow file:
+It's possible to facilitate GitHub Action's own email notification mechanism to send emails about Cirrus CI failures.Simply add the following worflow to `.github/workflows/email.yml` file:
```

Comment:
```suggestion
It's possible to facilitate GitHub Action's own email notification mechanism to send emails about Cirrus CI failures. Simply add the following worflow to `.github/workflows/email.yml` (or any existing workflow) file:
```

Reply:
IMO each workflow should be in a separate file 🤔